lib/matplotlib/axes/_axes.py

```diff
@@ -2691,78 +2691,84 @@ def errorbar(self, x, y, yerr=None, xerr=None,
                  barsabove=False, lolims=False, uplims=False,
                  xlolims=False, xuplims=False, errorevery=1, capthick=None,
                  **kwargs):
-        """
-        Plot an errorbar graph.
-
-        Call signature::
-
-          errorbar(x, y, yerr=None, xerr=None,
-                   fmt='', ecolor=None, elinewidth=None, capsize=None,
-                   barsabove=False, lolims=False, uplims=False,
-                   xlolims=False, xuplims=False, errorevery=1,
-                   capthick=None)
-
-        Plot *x* versus *y* with error deltas in *yerr* and *xerr*.
-        Vertical errorbars are plotted if *yerr* is not *None*.
-        Horizontal errorbars are plotted if *xerr* is not *None*.
+        """Plot an errorbar graph.
 
-        *x*, *y*, *xerr*, and *yerr* can all be scalars, which plots a
-        single error bar at *x*, *y*.
+        Plot x versus y with error deltas in yerr and xerr.
+        Vertical errorbars are plotted if yerr is not None.
+        Horizontal errorbars are plotted if xerr is not None.
 
-        Optional keyword arguments:
+        x, y, xerr, and yerr can all be scalars, which plots a
+        single error bar at x, y.
 
-          *xerr*/*yerr*: [ scalar | N, Nx1, or 2xN array-like ]
+        Parameters
+        ----------
+        x : scalar            
+        y : scalar
+            
+        xerr/yerr : scalar or array-like, shape(n,1) or shape(2,n), optional, default: None
```

Comment:
This line is too long for our code requirements (max 79 char). Can you remove the default and add it in the text below?